errors/no-async-client-component.mdx

```diff
@@ -6,9 +6,141 @@ title: No async client component
 
 ## Why This Error Occurred
 
-As per the [React Server Component RFC on promise support](https://github.com/acdlite/rfcs/blob/first-class-promises/text/0000-first-class-support-for-promises.md), [client components cannot be async functions](https://github.com/acdlite/rfcs/blob/first-class-promises/text/0000-first-class-support-for-promises.md#why-cant-client-components-be-async-functions).
+The error occurs when you try to define a client component as an async function. React client components [do not support](https://github.com/acdlite/rfcs/blob/first-class-promises/text/0000-first-class-support-for-promises.md#why-cant-client-components-be-async-functions) async functions. For example:
+
+```tsx
+'use client'
+
+// This will cause an error
+async function ClientComponent() {
+  // ...
+}
+```
 
 ## Possible Ways to Fix It
```

Comment:
Possible Solutions?

Reply:
We have this same format for all the error pages currently, but I agree that this heading is likely more clear. Maybe we do a pass and update them all 😄